### ISSUE TITLE
Product import - Image position

### ISSUE THREAD

**Issue by delyriand** [magento/magento2#7886]
### Preconditions
1. Magento CE 2.1.0

### Steps to reproduce
1. Create a product with sku T1234 without images.
2. Create a csv file that looks like (with less than an additional image):
```
sku;base_image;small_image;thumbnail_image;additional_images
T1234;tire_bouchon.jpg;tire_bouchon.jpg;tire_bouchon.jpg;tire_bouchon_2.jpg,tire_bouchon_3.jpg
```
3. Go to `System -> Import`` and import the file

### Expected result
1. Go to product view in front
2. Product images in this order: tire_bouchon.jpg, tire_bouchon_2.jpg and tire_bouchon_3.jpg

### Actual result
1. Go to product view in front
2. Currently the images are in wrong order: tire_bouchon_2.jpg, tire_bouchon.jpg and tire_bouchon_3.jpg.

---

This behavior is problematic on configurable products. Because when you choose a color, the image is not the right one

**Comment by nikoelgatito** [comment]
I found this bug report while looking for an other one. Actually this is very important for all product types including simple products. In several cases some pictures are way more attractive than others and should show up first while other images, are sometimes more technical or they are simple pictures showing the back of a box. Showing secondary images first can lead to loss of sales.

**Comment by pradeep-agnihotri** [comment]
This issue is still present in Magento 2.1.3, both Community & Enterprise.
It can be seen that the Additional Image and the Base Image both have same position "1" in the table 'catalog_product_entity_media_gallery_value'. 

Currently the only workaround is to update the value directly in the Database or via Admin Panel. There is not an option to specify the position during the import.

**Comment by joachimVT** [comment]
I'm having the same issue in Magento ver. 2.1.4

**Comment by PieterCappelle** [comment]
@sergiosemov Is it possible this is fixed in Develop branch > https://github.com/magento/magento2/commit/016b01097cbcdc71a2c83477ce448fb843afe1c0

**Comment by sergiosemov** [comment]
@PieterCappelle This issue is fixed in develop 2.2.0.

**Comment by slavvka** [comment]
The issue was fixed for 2.2 and delivered to mainline. Also it was fixed for 2.1 and will be delivered as a part of 2.1.6 Patch Release

**Comment by magento-engcom-team** [comment]
@delyriand, thank you for your report.
The issue is already fixed in develop branch, 2.2.0, 2.1.9
